Tests/MapboxMapsTests/Gestures/Mocks/MockGestureManagerDelegate.swift

```diff
@@ -1,8 +1,33 @@
 import MapboxMaps
 
 final class MockGestureManagerDelegate: GestureManagerDelegate {
-    let gestureBeganStub = Stub<GestureType, Void>()
-    func gestureBegan(for gestureType: GestureType) {
-        gestureBeganStub.call(with: gestureType)
+    struct GestureParams {
+        var gestureManager: GestureManager
+        var gestureType: GestureType
+    }
+
+    let gestureBeganStub = Stub<GestureParams, Void>()
+    func gestureManager(_ gestureManager: GestureManager, didBegin gestureType: GestureType) {
+        let parameters = GestureParams(gestureManager: gestureManager, gestureType: gestureType)
+        gestureBeganStub.call(with: parameters)
+    }
+
+    let animationEndedStub = Stub<GestureParams, Void>()
+    func gestureManager(_ gestureManager: GestureManager, didEndAnimatingFor gestureType: GestureType) {
+        let parameters = GestureParams(gestureManager: gestureManager, gestureType: gestureType)
+        animationEndedStub.call(with: parameters)
+    }
+
+    struct GestureEndedParams {
+        var gestureManager: GestureManager
+        var gestureType: GestureType
+        var willAnimate: Bool
+    }
+    let gestureEndedStub = Stub<GestureEndedParams, Void>()
```

Comment:
Let's update the names here for more consistency:

* Change `gestureBeganStub` to `gestureDidBeginStub` to match the new method signature
* Change `animationEndedStub` to `gestureDidEndAnimatingStub`
* Change `gestureEndedStub` to `gestureDidEndStub`
* Change `GestureEndedParams` to `GestureDidEndParams`